docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+x-env_file:
+  &env-file
+  env_file:
+    - templated.env
+x-network:
+  &network
+  networks:
+    - postgres-network
+x-restart-policy:
+  &restart-policy
+  restart: "no"
+
+version: '3.8'
+services:
+  database:
+    <<: *env-file
+    <<: *network
+    <<: *restart-policy
+    image: postgres:15.1
+    ports:
+      - "$DATABASE_PORT:5432"
+    volumes:
+      - postgres:/var/lib/postgresql/data
+  bot:
```

Comment:
If this is using pm2 internally, you probably want to store the logs in a volume so they dont get lost when you rebuild the container.
```
volumes:
  - pm2-logs:/app/logs
```